boards/t420/t420.config

```diff
@@ -0,0 +1,30 @@
+# Configuration for a x220 running Qubes and other OS
+# The Linux configuration is close enough to the x230
+export CONFIG_COREBOOT=y
+CONFIG_COREBOOT_CONFIG=config/coreboot-x220.config
+CONFIG_LINUX_CONFIG=config/linux-x230.config
```

Comment:
if correct, we should add a comment, why, I think

Reply:
This is correct. If the already existing X220 is close enough to X230, and T420 is close enough to X220, T420 is close enough to X230. 

Tested on my T420 as working.